tools/packaging/packaging.psm1

```diff
@@ -1229,85 +1215,554 @@ function New-ZipPackage
     }
 }
 
-function New-NugetPackage
```

Comment:
We need to be able to build both kinds of nuget package.  This is used for updating to the latest daily build.

Reply:
Added back.